frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard-steps/select-capacity-and-nodes-step.tsx

```diff
@@ -42,6 +59,110 @@ const validate = (scName, enableMinimal, enableFlexibleScaling): ValidationType[
   return validations;
 };
 
+const isArbiterDisabled = (nodesData: NodeKind[], validateSelectedNodes?: boolean) => {
+  const validation = {
+    nodesLen: validateSelectedNodes ? 4 : 5,
+    zoneLen: validateSelectedNodes ? 2 : 3,
+  };
+  const uniqZones: Set<string> = new Set(nodesData.map((node) => getZone(node)));
+  if (nodesData.length < validation.nodesLen) return true;
```

Comment:
```suggestion
  if (nodesData.length < validation.nodesLen) return true;
  const uniqZones: Set<string> = new Set(nodesData.map((node) => getZone(node)));
```
We can return early.